.luacheckrc

```diff
@@ -1,6 +1,4 @@
 include_files = {"**/*.lua", "*.rockspec", "*.luacheckrc"}
```

Comment:
Nit: It seems, we should adjust the wording in the changelog a bit.

Reply:
Fixed.

Reply:
Shouldn't we say about `:exec()` call, not just 'server instance'?

Reply:
Fixed.